test/int/featureBasedSuits/hitCountBreakpointsDifferentOperators.test.ts

```diff
@@ -63,23 +64,22 @@ puppeteerSuite('Hit count breakpoints combinations', reactWithLoopTestSpecificat
 
             const buttonClicked = incBtn.click();
 
-            if (conditionConfiguration.iterationsExpectedToPause.length > 0) {
-                for (const nextIterationToPause of conditionConfiguration.iterationsExpectedToPause) {
-                    /**
-                     * The iterationNumber variable counts in the js-debuggee code how many times the loop was executed. We verify
-                     * the value of this variable to validate that a bp with = 12 paused on the 12th iteration rather than on the 1st one
-                     * (The breakpoint is located in the same place in both iterations, so we need to use state to differenciate between those two cases)
-                     */
-                    await setStateBreakpoint.assertIsHitThenResume({ variables: { 'iterationNumber': `${nextIterationToPause}` } });
-                }
+            for (const nextIterationToPause of conditionConfiguration.iterationsExpectedToPause) {
+                /**
+                 * The iterationNumber variable counts in the js-debuggee code how many times the loop was executed. We verify
+                 * the value of this variable to validate that a bp with = 12 paused on the 12th iteration rather than on the 1st one
+                 * (The breakpoint is located in the same place in both iterations, so we need to use state to differenciate between those two cases)
+                 */
+                await setStateBreakpoint.assertIsHitThenResume({ variables: { 'iterationNumber': `${nextIterationToPause}` } });
+            }
 
+            if (conditionConfiguration.noMorePausesAfterwards) {
+                await breakpoints.assertNotPaused();
```

Comment:
I'm curious how this works, is it "the app is not paused right now" or "the app doesn't pause within the next x ms" or something else?

Reply:
Oh it's in this PR. You might rename it to `noPauseWithin1s` or something to that effect?

Reply:
Changed to waitAndAssertNotPaused(millisecondsToWaitForPauses = 1000 /*ms*/). What do you think?

I'd rather not put the exact time on the method, because we might need to change it for the build server...

Reply:
Yeah that sounds perfect.